apps/condo/domains/acquiring/utils/serverSchema/feeDistribution.js

```diff
@@ -8,17 +8,58 @@
     γ - commission from initial amount of payment, 0.004 ( 0.4 %)
 */
 
-/*  Fee settings
+/*
+    Differences between service fee and commission
+    Commission:
+        1. Is calculated based on the billingReceipt amount. For $1000 with commission 1.2% - commission will be $12
+        2. Can have minimum and maximum amount
+    Service fee:
+        1. Is calculated based on formula ↑. Because, acquiring get commission based on total payment (billingReceipt + serviceFee)
```

Comment:
It's hard to understand
Could you please explain calculation in order.
Like
Amount = 
Comission =
Service fee =
Acquiring fee =
Additional amount =